Project.toml

```diff
@@ -1,6 +1,7 @@
 [deps]
 Franklin = "713c75ef-9fc9-4b05-94a9-213340da978e"
 NodeJS = "2bd173c7-0d6d-553b-b6af-13a54713934c"
+Xranklin = "558449b0-171e-4e1f-900f-d076a5ddf486"
```

Comment:
Ooops

Reply:
Or, if it refers to https://github.com/tlienart/Xranklin.jl I you need to check in the manifest (which for the website would be a good idea)

Reply:
Follow-up here: https://github.com/JuliaParallel/juliaparallel.github.io/pull/4